drivers/mct/ocn_import_export.F90

```diff
@@ -237,26 +237,29 @@ subroutine ocn_import(x2o, ldiag_cpl, errorCode)
          !***  converting from m**2/s**2 to cm**2/s**2
          WORKB(i,j       ) = x2o(index_x2o_So_duu10n,n)
          U10_SQR(i,j,iblock) = cmperm * cmperm * WORKB(i,j) * RCALCT(i,j,iblock)
-         
-         ! QL, 150526, langmuir mixing variables
-         ! QL, 150908, only apply enhancement factor to ice-free region
+
+         ! langmuir mixing variables
+         ! apply enhancement factor to ice-free region
          if (IFRAC(i,j,iblock) <= 0.05_r8) then
             ! import enhancement factor (unitless)
             WORKB(i,j       ) = x2o(index_x2o_Sw_lamult,n)
             LAMULT(i,j,iblock) = WORKB(i,j)*RCALCT(i,j,iblock)
          else
             LAMULT(i,j,iblock) = c1
          endif
-         ! QL, 150706, DEBUG
-         !print*, "LAMULT = ", LAMULT(i,j,iblock)  
-         !print*, "RCALCT = ", RCALCT(i,j,iblock)  
-         !print*, "WORKB = ", WORKB(i,j)  
-         !! DEBUG
          ! import surface Stokes drift (m/s)
          WORKB(i,j       ) = x2o(index_x2o_Sw_ustokes,n)
          USTOKES(i,j,iblock) = WORKB(i,j)*RCALCT(i,j,iblock)
          WORKB(i,j       ) = x2o(index_x2o_Sw_vstokes,n)
          VSTOKES(i,j,iblock) = WORKB(i,j)*RCALCT(i,j,iblock)
+         ! apply surface layer Langmuir number to ice-free region
+         if (IFRAC(i,j,iblock) <= 0.05_r8) then
+            ! import surface layer Langmuir number (unitless)
+            WORKB(i,j       ) = x2o(index_x2o_Sw_hstokes,n)
+            LASL(i,j,iblock) = WORKB(i,j)*RCALCT(i,j,iblock)
+         else
+            LASL(i,j,iblock) = bignum
```

Comment:
I know I recommended `bignum` instead of `c10000`, but if `LASL` must be positive (per [vmix_kpp.F90#L2509](https://github.com/ESCOMP/POP2-CESM/blob/b029469f2a4bcf7281100afdb6209a46318030ec/source/vmix_kpp.F90#L2509)), does it make more sense to set `LASL(i, j, iblock) = -c1` here instead of using `bignum`?

Reply:
Yes, this is better. Then I will not need to check `IFRAC` again in https://github.com/ESCOMP/POP2-CESM/blob/b029469f2a4bcf7281100afdb6209a46318030ec/source/vmix_kpp.F90#L2509

I will do that. Thanks.